numpy/core/include/numpy/npy_1_7_deprecated_api.h

```diff
@@ -46,7 +46,7 @@
 #define NPY_ALIGNED        NPY_ARRAY_ALIGNED
 #define NPY_NOTSWAPPED     NPY_ARRAY_NOTSWAPPED
 #define NPY_WRITEABLE      NPY_ARRAY_WRITEABLE
-#define NPY_UPDATEIFCOPY   NPY_ARRAY_UPDATEIFCOPY
+#define NPY_UPDATEIFCOPY   NPY_ARRAY_UPDATEIFCOPY_CLEAR_B4_EXIT
```

Comment:
This should be reverted -- if someone's using the deprecated `NPY_UPDATEIFCOPY` alias, then they should get `NPY_ARRAY_UPDATEIFCOPY` semantics.